docs/source/examples/pec_tutorial.myst

```diff
@@ -0,0 +1,310 @@
+        if is_noisy:            
+            circuit_to_run.apply_gate_noise(
+                Noise.Depolarizing(probability=cnot_error_prob/2), target_gates=gates.CNot
+            )
+        to_run.append(circuit_to_run)
+        
+    # Process each job.
+    results: List[float] = []
+    for circ_to_run in to_run:
+        job_result = backend.run(circ_to_run, shots=shots).result()
+        results.append(job_result.measurement_probabilities.get("".join(map(str, correct_bitstring)), 0.0))
+
+    return results
+```
+
+## Running the experiment
+
+PEC samples many circuits according to the input representations. For each unitary, $U$ in the circuit with representation
+
+$$
+U = \sum_i a_i G_i,
+$$
+where channel $G_i$ replaces $U$ with (normalized) probability $|a_i|$.
+
+```{code-cell} ipython3
+cnot_counts, oneq_counts = [], []
+true_values, noisy_values = [], []
+pec_values = []
+
+noise_scaled_expectation_values = []
+
+for depth in depths:
+    print("Status: On depth", depth, end="\n\n")
+    
+    cnot_depth_counts, oneq_depth_counts = [], []
+    true_depth_values, noisy_depth_values = [], []
+    pec_depth_values = []
+
+    noise_scaled_expectation_depth_values = []
+
+    for trial in range(trials):
+        # Local seed is calculated in this way to ensure that we don't get repeat values in loop.        
+        local_seed = 10**6 * depth + 10**3 * seed + trial
+        
+        circuit, correct_bitstring = get_circuit(depth, local_seed)
+
+        true_value, = execute(circuit, ideal_backend, shots, correct_bitstring, is_noisy=False)
+        noisy_value, = execute(circuit, noisy_backend, shots, correct_bitstring, is_noisy=True)
+        
+        pec_executor = functools.partial(
+            execute, 
+            backend=noisy_backend, 
+            shots=shots // num_samples,
+            correct_bitstring=correct_bitstring,
+        )
+        
+        pec_value = pec.execute_with_pec(
+            circuit, 
+            pec_executor, 
+            representations=get_representations(computer), 
+            num_samples=num_samples,
+            random_state=local_seed,
+        )
+
+        pec_depth_values.append(pec_value)
+
+        cnot_depth_counts.append(get_num_cnot_count(circuit))
+        oneq_depth_counts.append(get_oneq_count(circuit))
+        
+        true_depth_values.append(true_value)
+        noisy_depth_values.append(noisy_value)
+    
+    cnot_counts.append(cnot_depth_counts)
+    oneq_counts.append(oneq_depth_counts)
+    
+    true_values.append(true_depth_values)
+    noisy_values.append(noisy_depth_values)
+    
+    noise_scaled_expectation_values.append(noise_scaled_expectation_depth_values)    
+    
+    pec_values.append(pec_depth_values)
+```
+
+For each call of `execute`, we have a collection of expectation values and signs. We combine them as per
+
+$$
+\frac{\eta}{M} \sum_{i=1}^M \sigma_i E_i
+$$
+
+which is an unbiased estimator of the true (noiseless) expectation value.
```

Comment:
```suggestion
```
At this stage of the tutorial, PEC is already done and noisy expectation values combined to get the mitigated results. At this stage we just need to plot them.

So I moved this part in my previous code-suggestion, with small changes.